### DISCUSSION TITLE
Implement drag and drop capabilities

### DISCUSSION THREAD

**Discussion by Aarbel** [TanStack/virtual#297] [Ideas]
Thanks a lot @tannerlinsley for this amazing lib.

Do you plan to support drag and drop in virtualized list soon ?
Or does this lib works well with a particular drag and drop library ?

We'd like to use your library to perform files / folders drag and drop in a file tree ui.

Thanks a lot for your help

**Comment by amitsingh-007** [comment]
I also have a similar use case where we want to use dnd in a file system UI. I tried using it with [use-gesture](https://github.com/pmndrs/use-gesture) but could not make it work.
